Apps/WebClient/src/Server/Services/UserSMSService.cs

```diff
@@ -63,6 +65,8 @@ public bool ValidateSMS(string hdid, string validationCode, string bearerToken)
             if (smsInvite != null &&
                 smsInvite.HdId == hdid &&
                 !smsInvite.Validated &&
+                !smsInvite.Deleted &&
+                smsInvite.VerificationAttempts < MaxVerificationAttempts &&
```

Comment:
Do the verification attempts ever get reset?

Reply:
Do we need the deleted?

Reply:
Verification attempts do not reset, once its past 5 tries the email/sms is gone and the user have to reenter it.

Reply:
We need the deleted for when the user removes their email/sms.